src/components/academies/AcademyDetailPage.jsx

```diff
@@ -33,6 +37,32 @@ const AcademyDetailPage = () => {
     },
     [config.ALGOLIA_APP_ID, config.ALGOLIA_INDEX_NAME, config.ALGOLIA_SEARCH_API_KEY],
   );
+  useEffect(() => {
+    const fetchPathway = async () => {
```

Comment:
I think using a custom hook here would be better and if possible we should make use of React Query as demonstrated by team titans